Remotion/Data/DomainObjects.UnitTests/IntegrationTests/Unload/UnloadDataTest.cs

```diff
@@ -129,7 +129,12 @@ public void UnloadData_Computer_WithUnsynchronizedOppositeEndPoint ()
       employee.EnsureDataAvailable();
 
       var unsynchronizedComputerID =
-          RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction<Computer, Employee>(employee.ID, (ot, o) => ot.Employee = o);
+          RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction<Computer, Employee>(employee.ID, (ot, o) =>
+          {
+            ot.SerialNumber = "12345";
+            o.Name = "Employee";
```

Comment:
please move those to lines 127ff.
This delegate is just for the relation assignment and we shouldn't muddle the waters to keep the intended effects easily understandable.